### DISCUSSION TITLE
General random discussion [OFF-TOPIC]

### DISCUSSION THREAD

**Discussion by yPiuro** [anno-org/NotesRUs#88] [General]
# This is a discussion to speak with each other during school time

**Comment by yPiuro** [comment]
Cant turn off EFI boot so cant use ISO without thumbdrive

**Reply by Notliam99** [reply]
you have a pizza box in your bag

**Reply by Notliam99** [reply]
WTF

**Comment by Notliam99** [comment]
HAHAHAHAHAHAHAHA
